projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/checkout/delivery-mode.ts

```diff
@@ -1,23 +1,22 @@
 import { checkoutNextStep, verifyTabbingOrder } from '../../tabbing-order';
-import { user } from '../../../../sample-data/checkout-flow';
 import { TabElement } from '../../tabbing-order.model';
 
-const containerSelector = '.MultiStepCheckoutSummaryPageTemplate';
+const containerSelector = 'cx-page-layout.MultiStepCheckoutSummaryPageTemplate';
 
 export function checkoutDeliveryModeTabbingOrder(config: TabElement[]) {
-  cy.window().then((win) => {
-    const { auth } = JSON.parse(
-      win.localStorage.getItem('spartacus-local-data')
-    );
-    cy.requireProductAddedToCart(auth).then(() => {
-      cy.requireShippingAddressAdded(user.address, auth);
-    });
-  });
-
   cy.visit('/checkout/delivery-mode');
 
   cy.get('cx-delivery-mode input');
 
   verifyTabbingOrder(containerSelector, config);
   checkoutNextStep('/checkout/payment-details');
 }
+
+export function checkoutDeliveryModeTabbingOrderAccount(config: TabElement[]) {
+  cy.visit('/checkout/delivery-mode');
```

Comment:
this is failing at times because the page failed to load, I believe we need to add the `waitForPage` functionality to reinforce the statement and have cypress wait an additional time for it to render. 

Also, we need to add this assertion in other places as well as I noticed in the old tabbing order ways did not make use of the functionality, so it would be best to add in those instances under `/accessibility/tabbing-order/**/*all file with visit at the beginning of a function

Reply:
Agreed